### PR TITLE
.gitignore Sample Data (used by DDEV install)

### PR DESCRIPTION
**Pull request by sreichel**
k/t

### CODE REVIEW

**Comment by ADDISON74** [comment]
This PR also took over the modified files from another PR #3787, but we can approve them both. I will provide you my feedback.

File: .gitignore [hunk 1]
```diff
@@ -80,3 +80,6 @@ phpstan*.neon
 # dev scripts loaded via composer
 /shell/update-copyright.php
 /shell/translations.php
+
+# sample data
+magento_sample_data_for_1.9.2.4.sql
```
Comment (fballiano) [inline .gitignore:85]: my personal opinion I don't think we should have every possible file in the gitignore

Comment (ADDISON74) [reply .gitignore:85]: When we use the ```ddev openmage-install``` command we get in the root directory that file and it stays there from now on. It should be ignored by git.

Comment (fballiano) [reply .gitignore:85]: then maybe it should be deleted afterwards

Comment (ADDISON74) [reply .gitignore:85]: It is debatable. That file is good for creating a new database quickly, without having to download the archive from the Internet.

Comment (fballiano) [reply .gitignore:85]: git is also very good at showing which files are there and shouldn't be, which allows for a better housekeeping of every project. there more we add to the gitignore the less effective housekeeping is, imho. that's why my opinion is that we shouldn't have every possible file for every possible use in the gitignore, which imho again is already cluttered and could be simplified

Comment (sreichel) [reply .gitignore:85]: As @ADDISON74 said, it is used by ddev install. To keep it is optional to repeat installing sample data without download it every time.

**Comment by sreichel** [comment]
On hold ... install sample data can be improved.

**Pull request closed without merging**